shlr/meson.build

```diff
@@ -199,15 +199,15 @@ sdb_dep = declare_dependency(
 )
 
 # Create sdb_version.h
-sdb_version = '1.2.0'
+sdb_version = '1.5.0'
```

Comment:
sdb version string is not in sync in the sdb repo either.. what about defining a way to import this and why do you  need to generate this sdb_version.h ? this should be done by SDB iitself in shlr/sdb not in r2

Reply:
yes, it should be done by sdb, but to do we need proper using of meson in both sdb and r2 with subprojects and such. The file is generated because also the shlr/sdb/Makefile generates it. We could extract the sdb_version value from shlr/sdb/config.mk istead of hardcoding it here.